lib/src/__tests__/typescript/GenericValues.tstest.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as React from 'react';
```

Comment:
One question, how are these tests run in the CI, I can't find where it comes from.